NetKAN/MusicMuteContinued.netkan

```diff
@@ -1,13 +1,16 @@
 {
-    "spec_version" : "v1.4",
-    "identifier"   : "MusicMuteContinued",
-    "$kref"        : "#/ckan/github/nimobeeren/MusicMute",
-    "name"         : "MusicMute",
-    "abstract"     : "A simple mod that toggles the game's music with a hotkey.",
-    "license"      : "GPL-3.0",
-    "ksp_version"  : "1.2",
+    "spec_version"       : "v1.16",
+    "identifier"         : "MusicMuteContinued",
+    "$kref"              : "#/ckan/github/nimobeeren/MusicMute",
+    "name"               : "MusicMute",
+    "abstract"           : "A simple mod that toggles the game's music with a hotkey.",
+    "license"            : "GPL-3.0",
+    "ksp_version_min"    : "1.2",
+    "ksp_version_max"    : "1.3",
+    "ksp_version_strict" : "false",
     "resources": {
```

Comment:
Wrapping `"false"` in quotes turns it into a string of text. The schema expects `true` or `false` in that field.

```
Validating built/MusicMuteContinued-1.1.3.ckan.. Failed! See below for error description.
u'false' is not of type u'boolean'

Failed validating u'type' in schema[u'properties'][u'ksp_version_strict']:
    {u'description': u'Optional enforcement of strict version checks (defaults to false)',
     u'type': u'boolean'}

On instance[u'ksp_version_strict']:
    u'false'
```